components/user_settings/modal/user_settings_modal.jsx

```diff
@@ -150,7 +152,7 @@ class UserSettingsModal extends React.Component {
 
     // Called to hide the settings pane when on mobile
     handleCollapse = () => {
-        $(ReactDOM.findDOMNode(this.refs.modalBody)).closest('.modal-dialog').removeClass('display--content');
+        $(ReactDOM.findDOMNode(this.modalBodyRef.current)).closest('.modal-dialog').removeClass('display--content');
```

Comment:
same here

Reply:
The closest is not so easy, is not supported in IE11, so... you may need use a polyfill or achieve the same behavior using other approach.

Reply:
Anyway, removing the jquery can be something that we can do in the future, we don't need it right now.